components/molecule/field/src/index.js

```diff
@@ -11,60 +11,51 @@ import AtomHelpText from '@s-ui/react-atom-help-text'
 const BASE_CLASS = 'sui-MoleculeField'
 const CLASS_CONTAINER = `${BASE_CLASS}-inputContainer`
 
-class MoleculeField extends Component {
-  get className() {
-    const {inline} = this.props
-    return cx(BASE_CLASS, inline && `${BASE_CLASS}--inline`)
+const MoleculeField = ({
+  inline,
+  errorText,
+  successText,
+  label,
+  helpText,
+  name,
+  onClickLabel,
+  children // eslint-disable-line
```

Comment:
Instead disabling it, we should add `children` as a propType.

Reply:
done!